docs/source/benchmarking.rst

```diff
@@ -13,66 +13,10 @@ Benchmarking
 Benchmarking with the ``autometa-benchmark`` module
 ===================================================
 
-Example benchmarking with simulated communities
------------------------------------------------
-
-Download all of the simulated communities and their reference assignments
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-.. code:: bash
-
-    community_sizes=(78Mbp 156Mbp 312Mbp 625Mbp 1250Mbp 2500Mbp 5000Mbp 10000Mbp)
-    autometa-download-dataset \
-        --community-type simulated \
-        --community-sizes ${community_sizes[@]} \
-        --file-names reference_assignments.tsv.gz binning.tsv.gz taxonomy.tsv.gz \
-        --dir-path simulated
-
-Benchmark all of the simulated communities and their reference assignments
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-.. code:: bash
-
-    for community_size in ${community_sizes[@]};do
-        autometa-benchmark \
-            --benchmark clustering \
-            --predictions simulated/${community_size}/binning.tsv.gz \
-            --reference simulated/${community_size}/reference_assignments.tsv.gz \
-            --output-wide ${community_size}.clustering_benchmarks.wide.tsv.gz \
-            --output-long ${community_size}.clustering_benchmarks.long.tsv.gz
-    done
-
-Aggregate across simulated communities (when dataset index is unique)
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-.. code:: python
-
-    import pandas as pd
-    import glob
-    df = pd.concat([
-        pd.read_csv(fp, sep="\t", index_col="dataset")
-        for fp in glob.glob("*.clustering_benchmarks.long.tsv.gz")
-    ])
-    df.to_csv("benchmarks.tsv", sep='\t', index=True, header=True)
-
-Aggregate across simulated communities (when dataset index is `not` unique)
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-.. code:: python
-
-    import pandas as pd
-    import os
-    import glob
-    dfs = []
-    for fp in glob.glob("*.clustering_benchmarks.long.tsv.gz"):
-        df = pd.read_csv(fp, sep="\t", index_col="dataset")
-        df.index = df.index.map(lambda fpath: os.path.basename(fpath))
-        dfs.append(df)
-    df = pd.concat(dfs)
-    df.to_csv("benchmarks.tsv", sep='\t', index=True, header=True)
```

Comment:
I don't think the code to aggregate benchmarking results should be removed from the documentation. This is a reference for anyone wanting to be able to quickly merge all of their benchmarking results.

Reply:
I understand. 
I perharps was confused between the two cases. I ran both of the code chunks (when index is unique and not unique) for `625Mbp.clustering_benchmarks.long.tsv.gz` and `78Mbp.clustering_benchmarks.long.tsv.gz` and git the same results.
Can you please elaborate on the differences between the two and when the user should chose which.

Reply:
If you inspect the python code, you will see the generated benchmarks are being read into pandas dataframes and the index is being set to "dataset". e.g. `pd.read_csv(..., index_col="dataset")`. You will also notice that if you know the "dataset" values are non-unique for your generated benchmarking files (you can find this out through manual inspection or by handling with python/R/bash) then the index (dataset value) will be renamed to the filename of the respective file. This was mentioned in https://github.com/KwanLab/Autometa/pull/141#issuecomment-847348201 so it is placed here as convenience for anyone who may need to aggregate multiple binning results (whether they have unique dataset values or not).